open-craft/opencraft
# SE-1617 update ecommerce docs

This updates the e-commerce installation docs for ocim.


**Test instructions**:

- Follow conversation on [SE-1617](https://tasks.opencraft.com/browse/SE-1617) and verify that updates here match what was done and the config on the client's instance.
- check that the markdown renders in github nicely.

**Reviewer**:
